CONTRIBUTING.md

```diff
@@ -1,32 +1,314 @@
+- `lerna exec <script>`: run script on all packages unless you ignore packages with `--ignore=<pkg1>,<pkg2>` or specify in which it should execute with `--scope=<pkg3>,<pkg4>`.
+- `lerna clean`: clean _node_modules_ from root and all packages.
+- `lerna create <name> <location>`: create a new package named name in location specified (it could be `core`, `plugins` or `tools`).
+- `lerna import <source_package> --dest=<location>`: import an external package from _source_package_ in the location specified (it could be `core`, `plugins` or `tools`).
+
+There are more commands and options well explained you can see in their [GitHub repository](https://github.com/lerna/lerna).
+
+#### Yarn Workspaces
+
+We delegate high-level managing commands to Lerna, but there are some commands you should know:
+
+- `yarn workspaces info`: show info and relations between packages in the workspace, and if a package dependency from the workspace is not updated.
+- `yarn workspace <package> <command>`: run command in the package specified.
+
+For more information, take a look at [`yarn workspace`](https://yarnpkg.com/lang/en/docs/cli/workspace) and [`yarn workspaces`](https://yarnpkg.com/en/docs/cli/workspaces) CLI docs.
+
+### Creating Packages
+
+To create a new package, you can create with:
+
+`lerna create <package_name> <scope_location>`
+
+And then, follow the next steps in the package recently created:
+
+1. Add MIT LICENSE file (using`npx license -o Verdaccio mit > LICENSE`) and complete README.
+2. Add the necessary tools files the project will require like `.gitignore`, `.eslintrc.json`, `tsconfig.json` or `.babelrc`. You can see other packages in the monorepo to know which tools are we using.
+3. Add all the necessary stuff in the `package.json`, like scripts or dependencies, taking care of script naming like `lint`, `lint:stage` or `test`.
+4. In case the package is scoped (like `@verdaccio/streams`), add next block in `package.json`:
+   ```json
+   "publishConfig": {
+     "access": "public"
+   }
+   ```
+5. Flat those `devDependencies` that aren't part of the monorepo to the root `package.json`, like `in-publish`, but not `devDependencies` like `@verdaccio/eslint-config`.
+6. Ensure everything is working correctly, both in package and with root scripts.
+
+### Importing Packages
+
+The steps for importing and creating packages differ a little bit. First, you need to have cloned the project to import in your local filesystem and then you need to import all the project git tree using next command:
+
+`lerna import <project_location> --dest=<scope_location>`
+
+If this process fails because there were merge conflicts, you must add `--flatten` option to the command above to flat history.
+
+Later, follow the next steps in the package recently imported:
+
+1. Remove unnecessary stuff, like CI settings, GitHub stuff, CONTRIBUTING, unnecessary things from .gitignore, devDependencies, scripts, etc.
```

Comment:
What do you think about providing a checklist of what should be removed, something like a specific checklist now that we have them in the root folder? Something like,
```
Review the list of files you are importing, notably:
* .git folder - should be removed
* .gitignore - please review against the main .gitignore file
* remove any build pipeline files you may have
* update devDependencies and Dependencies to hoist packages from the main package.json
* update scripts
```

Reply:
Nice, I will elaborate a complete checklist for each step to not leave things in the air :wink: